Source/Core/VideoCommon/CMakeLists.txt

```diff
@@ -44,6 +44,10 @@ set(SRCS	BoundingBox.cpp
 			XFStructs.cpp)
 set(LIBS core png)
 
+if(_M_ARM_64)
+	set(SRCS ${SRCS} VertexLoaderARM64.cpp)
+endif()
```

Comment:
I'd like "else" more, so it'd be clear that this is a switch statement for the cpu architecture